src/is.test.ts

```diff
@@ -33,16 +33,319 @@ import {
   isTypedArray*/
 } from './is';
 
+// isArray(value)
 describe('isArray(value)', () => {
   it('Should be a function', () => {
     expect(isArray).to.be.a('function');
   });
 
-  it('Should should return true when "value" is type any[]', () => {
-    expect(isArray([1, '2', true])).to.equal(true);
+  it('Should return true when "value" is type any[]', () => {
+    expect(isArray([])).to.equal(true);
+    expect(isArray([1, 2, 3])).to.equal(true);
+    expect(isArray(['a', 'b', 'c'])).to.equal(true);
+    expect(isArray([1, '2', true, {}])).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type any[]', () => {
+    expect(isArray('This is a string')).to.equal(false);
+    expect(isArray(100)).to.equal(false);
+    expect(isArray(undefined)).to.equal(false);
+    expect(isArray(null)).to.equal(false);
+    expect(isArray(true)).to.equal(false);
+    expect(isArray(false)).to.equal(false);
+    expect(isArray({})).to.equal(false);
+    expect(isArray()).to.equal(false);
+  });
+});
+
+// isBoolean(value)
+describe('isBoolean(value)', () => {
+  it('Should be a function', () => {
+    expect(isBoolean).to.be.a('function');
+  });
+
+  it('Should return true when "value" is type boolean', () => {
+    expect(isBoolean(true)).to.equal(true);
+    expect(isBoolean(false)).to.equal(true);
+  });
+
+  it('Should return false when "value" is not type boolean', () => {
+    expect(isBoolean('This is a string')).to.equal(false);
+    expect(isBoolean(100)).to.equal(false);
+    expect(isBoolean(undefined)).to.equal(false);
+    expect(isBoolean(null)).to.equal(false);
+    expect(isBoolean([])).to.equal(false);
+    expect(isBoolean({})).to.equal(false);
+    expect(isBoolean()).to.equal(false);
+  });
+});
+
+// isInfinity(value)
+describe('isInfinity(value)', () => {
+  it('Should be a function', () => {
+    expect(isInfinity).to.be.a('function');
+  });
+
+  it('Should return true when "value" is Infinity', () => {
+    expect(isInfinity(Infinity)).to.equal(true);
+  });
```

Comment:
Let's add checks for the various ways that you can get the value Infinity:

```suggestion
  it('Should return true when "value" is Infinity', () => {
    expect(isInfinity(Infinity)).to.equal(true);
    expect(isInfinity(-Infinity)).to.equal(true);
    expect(isInfinity(Number.POSITIVE_INFINITY)).to.equal(true);
    expect(isInfinity(Number.NEGATIVE_INFINITY)).to.equal(true);
    expect(isInfinity(Number.MAX_VALUE * 1.000000000000001)).to.equal(true);
  });
```

Reply:
Will do. Will also check for mathematical Infinity ( number /0).